run_tool_shed.sh

```diff
@@ -2,6 +2,13 @@
 
 cd `dirname $0`
 
+: ${GALAXY_VIRTUAL_ENV:=.venv}
+
+if [ -d "$GALAXY_VIRTUAL_ENV" ];
+then
+    . "$GALAXY_VIRTUAL_ENV/bin/activate"
+fi
+
```

Comment:
This code must go AFTER `./scripts/common_startup.sh`, which will create the `.venv` directory if it does not exist.